### DISCUSSION TITLE
Not able to find references to certain consts

### DISCUSSION THREAD

**Discussion by brijesh-amin** [typescript-language-server/typescript-language-server#757] [Q&A]
tslanguage server not able to find references

<img width="1034" alt="image" src="https://github.com/typescript-language-server/typescript-language-server/assets/8287486/03b1ce91-c49e-49f9-a1eb-1b17995e5a58">

However, when I try to go to the definition of that same const I am able to,

<img width="1044" alt="image" src="https://github.com/typescript-language-server/typescript-language-server/assets/8287486/da2c86aa-87ef-46c6-887f-d2530a1fb766">

**Comment by brijesh-amin** [comment]
May be related to https://github.com/typescript-language-server/typescript-language-server/issues/271

**Comment by rchl** [comment]
Probably a project configuration issue. Hard to say without more details (ideally project that reproduces).
